REQUIRE

```diff
@@ -1,5 +1,5 @@
-julia 0.4
```

Comment:
also in travis, if stevengj agrees about now (well, after #502 gets merged and tagged first) being an okay time to drop 0.4